kafka/conn.py

```diff
@@ -54,6 +54,23 @@ class SSLWantReadError(Exception):
     class SSLWantWriteError(Exception):
         pass
 
+# needed for SASL_GSSAPI authentication:
+try:
+    import gssapi
+    #from gssapi import raw
+except ImportError:
+    #no gssapi available, will disable gssapi mechanism
+    gssapi = None
+    #class GSSError(Exception):
+    #    pass
+
+try:
```

Comment:
cool. you could also combine w/ the above import try. I think we can safely assume that either both imports work, or both will fail.